### PR TITLE
Add initial contributing guide.

### LINKED ISSUE #42: Add contributing guide
The repo doesn't have a contributing guide yet. Follow other examples in other repos to define our first one.

### PR DESCRIPTION
**Pull request by tlcraft**
This PR adds an initial contributing guide to our repo. I followed the Atom guide as an example (https://github.com/atom/atom/blob/master/CONTRIBUTING.md).

Sample from VS Code:
image

Closes #42

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Contributing to the Kalispell Software Crafters GitHub Pages Website
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
```
Comment (tlcraft) [inline CONTRIBUTING.md:3]: I'm expecting GitHub will display these icons correctly but my VS Code screenshot didn't do that.

Comment (tlcraft) [reply CONTRIBUTING.md:3]: Looks like they work!
https://github.com/kalispell-software-crafters/kalispell-software-crafters.github.io/blob/feature/add-contribution-guide/CONTRIBUTING.md

Comment (TestDrivenBexten) [reply CONTRIBUTING.md:3]: Really like the contributing guide from Atom. I appreciate that you put a section in for our code of conduct, too. I approve the merge

**Review by TestDrivenBexten — APPROVED** [review]
Really like the contributing guide from Atom. I appreciate that you put a section in for our code of conduct, too. I approve the merge

**Pull request merged**